src/main/java/com/google/api/codegen/config/GapicInterfaceConfig.java

```diff
@@ -128,14 +132,19 @@ static GapicInterfaceConfig createInterfaceConfig(
               resourceNameConfigs,
               retryCodesConfig,
               retrySettingsDefinition.keySet());
-      methodConfigs = createMethodConfigs(methodConfigMap, interfaceConfigProto);
```

Comment:
What keeps this from being nullable?

Reply:
I'm not sure I understand the question. 
`createMethodConfigs` will return a non-null list. The way that the parameters to that function are created ensures that `null` won't be put into the map.

Reply:
Ok didn't notice that at first. Thanks